packages/tables/src/Concerns/CanSearchRecords.php

```diff
@@ -141,7 +149,7 @@ protected function applyGlobalSearchToTableQuery(Builder $query): Builder
      */
     public function getTableSearch()
     {
-        return $this->tableSearch;
+        return $this->tableSearch === null ? null : trim($this->tableSearch);
```

Comment:
```suggestion
        return filled($this->tableSearch) ? trim($this->tableSearch) : null;
```

Reply:
Done that, with a test. An empty string will be retrned now as a `null`. It cannot return an empty string. Also a string containing only whitespace.